src/include/mpir_comm.h

```diff
@@ -68,6 +68,27 @@ int MPIR_Comm_map_dup(struct MPIR_Comm *newcomm, struct MPIR_Comm *src_comm,
                       MPIR_Comm_map_dir_t dir);
 int MPIR_Comm_map_free(struct MPIR_Comm *comm);
 
+/* Communicator info hint */
+#define MPIR_COMM_HINT_TYPE_BOOL 0
+#define MPIR_COMM_HINT_TYPE_INT  1
+
+/* Communicator attr (bitmask)
+ * If local bitis set, the hint is local. Default (0) will require the hint value be
+ * the same across communicator.
```

Comment:
Is the part for checking "the same value" implemented yet?

Reply:
No. pushed off :)